ReactAndroid/src/main/java/com/facebook/react/views/toolbar/ReactToolbarManager.java

```diff
@@ -84,6 +85,15 @@ public void setTitle(ReactToolbar view, @Nullable String title) {
     view.setTitle(title);
   }
 
+  @ReactProp(name = "rtl")
+  public void setLayoutDirection(ReactToolbar view, Boolean flag) {
+    if (flag.equals(true)) {
+      view.setLayoutDirection(LayoutDirection.RTL);
+    } else {
+      view.setLayoutDirection(LayoutDirection.LTR);
```

Comment:
Oh, ok. I guess it's actually reversed when locale is set to RTL, so this should be fine. In that case you can also make this method take primitive `boolean` instead of `Boolean` and implement this method in single line

    view.setLayoutDirection(flag ? LayoutDirection.LTR : LayoutDirection.RTL)